src/views/statutory/Single.vue

```diff
@@ -11,6 +11,13 @@
       </div>
       <div class="tile is-parent">
         <article class="tile is-child is-info">
+          <div class="field is-grouped" v-if="duringAgora">
+            <a href="https://aegeeagora-streaming.westeurope.cloudapp.azure.com/" class="button is-danger is-fullwidth">
```

Comment:
would it be static or would it change every Agora?

Reply:
It's static, although we might want to get a nicer link for next Agora